main.cpp

```diff
@@ -52,85 +51,102 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       car
+//  action 1:   the car rolls forward
+car.rollForward();
+//  action 2:   the car consumes gas
+car.consumeGas();
+//  action 3:   the car wipes its own windows with wipers
+car.wipeWindow();
+
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       snake
+//  action 1:   the snake rolls over
+snake.rollOver();
+//  action 2:   the snake eats a mouse
+snake.eatMouse();
+//  action 3:   the snake wraps around an object
+snake.wrapObject();
+
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:       pikachu
+//  action 1:   pikachu says something
+pikachu.sayWord();
+//  action 2:   pikachu goes to the restroom
+pikachu.goToRestroom();
+//  action 3:   pikachu drinks water
+pikachu.drinkWater();
 
-//  action 3:
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       bunny
+//  action 1:   bunny does a 360
+bunny.do360();
+//  action 2:   bunny attacks
+bunny.attack();
+//  action 3:   bunny ground attacks from above using missiles fired from a helicopter
+bunny.helicopterAttackMissile
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       computer mouse
+//  action 1:   mouse sends click signal to computer
+computerMouse.sendClick();
+//  action 2:   sends scroll signal
+computerMouse.sendScroll();
+//  action 3:   mouse depletes battery
+computerMouse.depleteBattery();
+
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog eats 
+dog.eat();
+//  action 3:   the dog flies
+dog.fly();
+
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       cat
+//  action 1:   the cat drives a car
+cat.driveCar();
+//  action 2:   the cat drinks milk
+cat.milk();
+//  action 3:   the cat googles something
+cat.google();
+
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       mouse
+//  action 1:   the mouse jumps
+mouse.jump();
+//  action 2:   the mouse says hi
+mouse.sayHi();
+//  action 3:   the mouse thinks
+mouse.think();
+
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.